docs/installation/installation-using-docker-application-setup.md

```diff
@@ -16,6 +16,11 @@ docker exec -it shopsys-framework-php-fpm sh
 composer install
 ```
 
+If you install the application in production, you should install composer optimized
```

Comment:
If you are installing the application in production environment ? I think that this sounds better, but it is only a suggestion. You are missing full stop at the end of the sentence.